OpenRA.Mods.Common/Traits/ProximityCapturable.cs

```diff
@@ -153,7 +147,7 @@ void UpdateOwnership()
 			{
 				if (Info.MustBeClear)
 				{
-					var isClear = IsClear(Self, captor.Owner);
+					var isClear = actorsInRange.All(a => a.Owner.RelationshipWith(captor.Owner) == PlayerRelationship.Ally);
```

Comment:
Shouldn't this be `captor.Owner.RelationshipWith(a.Owner)`?

Reply:
Derp, yes.

Reply:
Fixed again.